Client/Frontend/Browser/Tab.swift

```diff
@@ -275,10 +285,9 @@ class Tab: NSObject {
     deinit {
         deleteWebView()
         contentScriptManager.helpers.removeAll()
-        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
-        
-        let rewards = appDelegate.browserViewController.rewards
-        
+        guard let rewards = self.rewards else {
```

Comment:
No need to guard check this, can just use optional below:

`rewards?.reportTabClosed...`